messybrainz/fetch_artist_mbids.py

```diff
@@ -0,0 +1,117 @@
+# Script to fetch artist MBIDs from MusicBrainz Database using
+# the recording MBIDs in the MessyBrainz database.
+
+import brainzutils.musicbrainz_db.recording as mb_recording
+import json
+import logging
+import re
+
+import messybrainz.default_config as config
+try:
+    import messybrainz.custom_config as config
+except ImportError:
+    pass
+
+from brainzutils import cache
+from brainzutils import musicbrainz_db
+from brainzutils.musicbrainz_db.exceptions import NoDataFoundException
+from messybrainz import db
+from sqlalchemy import text
+
+
+def check_valid_uuid(s):
```

Comment:
In other projects, we just create a `uuid.UUID` object and return False if it raises a ValueError. I think that is a better method when compared to this long regex.

See [here](https://github.com/metabrainz/listenbrainz-server/blob/35313a656d781028d72e48cfd1500066067c08da/listenbrainz/webserver/views/api_tools.py#L147)

Reply:
+1

Never reinvent the wheel if you don't have to -- that means you don't have to debug the wheel when inevitable bugs appear.